### PR TITLE
docs(blog): development update june

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary of changes
- Added 2nd NOTAM and associated (draft) copy.
- Reference #media-team on Discord for any updates to the copy and the associated google docs link.
- Future suggestions should probably remain here on the PR from now on

## Upcoming Article(s) & Expected Due Date
- After Release of flyPadOS v2

## Article Discussion
- Add/change image for the post
- Google docs link has additional content that can / may be included in the post such as (*featured media, developer insights video).
    - One example is the Aileron Droop video.
    
## TO-DO

- [ ] Ensure Correct Dates and Title
- [x] Add credits to flyPadOSv2 section

## Progress of Article
- [x] TBA
- [x] Planning
- [x] Draft
- [ ] Review
- [ ] Posted

 [Preview - Direct Page Link](https://marketing-website-git-fork-valastiri-notam2-flybywire.vercel.app/notams/development_notam-2021-07-13/)

### CODE REVIEW

**Comment by Valastiri** [comment]
Reminder to alter "posted date" before merging

File: src/posts/development_notam-2021-06-20.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  - 'Boris'
+  - 'Kevin'
+category: 'DEVELOPMENT UPDATE'
+metaImage: '/img/notam-images/efb-featured.png'
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Overview
+
+It has been an amazing journey over these past ten months here at FlyByWire Simulations. We have some exciting things that are released on our development branch! In this NOTAM we are going to provide you with an overview of changes we have implemented towards the following:
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:17]:
```suggestion
It has been an amazing journey over these past ten months here at FlyByWire Simulations. We have some exciting things that are on our development branch! In this NOTAM we are going to give you an overview of changes that have been implemented:
```

Comment (NathanInnes) [reply src/posts/development_notam-2021-06-20.md:17]: General shortning of words that don't need to be complex for readability.

File: src/posts/development_notam-2021-06-20.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+- flyPadOS v2! Brand new UI and features available.
+- Sound implementation + upcoming planned audio effects
+- Hydraulics implementation and state
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:21]:
```suggestion
- Sound implementation + upcoming planned audio effects.
- Hydraulics implementation and state.
```
All items in a list should have a period to end it.

File: src/posts/development_notam-2021-06-20.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+---
+
+## Development Updates
+
+### flyPadOS v2
+
+The latest version of our flyPad has the following main features:
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:29]:
```suggestion
The latest update of our flyPad has has recieved:
```

File: src/posts/development_notam-2021-06-20.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+- Landing performance calculator
+- New user interface design
+- More options in the settings page
+- Navigraph charts integration
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:34]:
```suggestion
- Landing performance calculator.
- New user interface design.
- More options in the settings page.
- Navigraph charts integration.
```

File: src/posts/development_notam-2021-06-20.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+You heard it right! We’re excited to deliver Navigraph charts functionality in the EFB. Getting started is simple - all you need to do is to:
+
+1. Select the Charts tab in the EFB
+2. Scan the QR code
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:39]:
```suggestion
1. Select the Charts tab on the EFB.
2. Scan the QR code to authenticate.
```

File: src/posts/development_notam-2021-06-20.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+NOTE: The use of Navigraph charts requires an active Ultimate subscription with Navigraph. You can check the current price at [Navigraph Subscriptions](https://navigraph.com/products/subscriptions).
+
+We are also planning to add Chartfox integration to the EFB some time later.
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:43]:
```suggestion
We are also planning to add Chartfox integration to the EFB at a later date.
```

File: src/posts/development_notam-2021-06-20.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+As briefly touched on before, we have done a major redesign to the look and feel of the entire EFB for a more refined appearance and improved usability.
+
+While exploring the EFB you may have noticed that the settings page is now no longer greyed out. We have moved all the settings from the MCDU into the EFB. On a final note, the default keybinds no longer trigger when using an input while utilizing the EFB natively, meaning you no longer have to “pop-out” the EFB window.
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:47]:
```suggestion
While exploring the EFB you may have noticed that the settings page is now no longer greyed out. We have moved all the settings from the MCDU into the EFB. On a final note, the default keybinds no longer trigger when using an input while using the EFB, meaning you no longer have to “pop-out” the EFB window to type.
```

File: src/posts/development_notam-2021-06-20.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+
+[Credit to the developers who worked on FPOSv2]
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:50]: Can we find out who was involved so they can have more credibility?

File: src/posts/development_notam-2021-06-20.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+We recently merged major improvements to our sounds module in the A32NX. 
+
+Available now:
+
+- New avionics fans audio for a more immersive power up and shutdown.
+- Enhanced engine spool up
+- Reworked engine sounds from idle to TOGA (heard externally and internally)
+- Yellow and blue electric pumps sounds added (audible in the cabin mostly)
+- Sound of the ground service bus (everytime you load in you will hear a clunk and a small whine)
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:62]:
```suggestion
- Enhanced engine spool up.
- Reworked engine sounds from idle to TOGA (heard externally and internally).
- Yellow and blue electric pumps sounds added (audible in the cabin mostly).
- Sound of the ground service bus (everytime you load in you will hear a clunk and a small whine).
```

File: src/posts/development_notam-2021-06-20.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+Not only have we pushed noticeable changes that you can hear, but we’ve added in events that allow our developers to implement the logic at a later time.
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:64]:
```suggestion
Not only have we pushed noticeable changes that you can hear, but we’ve added in events that allow developers to implement the logic at a later time.
```

File: src/posts/development_notam-2021-06-20.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+
+Events added without logic:
+
+- TCAS wwise events
+- ROPS wwise events
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:69]:
```suggestion
- TCAS wwise events.
- ROPS wwise events.
```

File: src/posts/development_notam-2021-06-20.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+The last hydraulics update connected the brakes to MSFS physics engine, so the brakes will now only stop the plane if there is hydraulic pressure. However, the A32NX still uses the default autobrake system, which can bypass our implementation. Thus, our current priority is to develop a custom autobrake system to match all the conditions and dynamic deceleration targets of the real A320neo. 
+
+There are so many aspects of the hydraulic system to implement that we can’t cover them all here, but some of the next steps are:
+
+- Tuning brake forces to match real plane performance figures
+- Modelling the blue hydraulic/electric emergency generator
+- Modelling the RAT accurately
+- Creating a physical model of the PTU
+- Connecting and modelling all actuators one by one
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:81]:
```suggestion
- Tuning brake forces to match real plane performance figures.
- Modelling the blue hydraulic/electric emergency generator.
- Modelling the RAT accurately.
- Creating a physical model of the PTU.
- Connecting and modelling all actuators one by one.
```

File: src/posts/development_notam-2021-06-20.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+- Ensure that your issue has not already been reported on our GitHub.
+
+Download the latest version of the A32NX:
+
+- [Installer](https://api.flybywiresim.com/installer)
+- [Standalone Versions](https://flybywiresim.com/a32nx/#download)
+
+---
+
+Thanks for tuning in!
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:110]:
```suggestion
```

**Review by NathanInnes — COMMENTED** [review]
A few gramatical changes and some puncuation to look at for you @Valastiri

File: src/posts/development_notam-2021-06-20.md (hunk 8 above)
Comment (Valastiri) [reply src/posts/development_notam-2021-06-20.md:50]: This is pending till I get more information, since this portion was drafted by Falcon.

**Comment by Benjozork** [comment]
I'll have cFPM information soon.

File: src/posts/development_notam-2021-06-20.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
+---
+title: 'Development Updates June 2021'
+date: '2021-06-20'
```
Comment (NathanInnes) [inline src/posts/development_notam-2021-06-20.md:3]: REMINDER: Update date upon ready to merge

**Comment by NathanInnes** [comment]
@Valastiri You think we should put the latest yt video in there?

File: src/posts/development_notam-2021-06-20.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+authors:
+  - 'Valastiri'
```
Comment (Benjozork) [inline src/posts/development_notam-2021-06-20.md:5]: Didn't we decide on using real names?

File: src/posts/development_notam-2021-06-20.md (hunk 14 above)
Comment (Valastiri) [reply src/posts/development_notam-2021-06-20.md:3]: added to original comment to track. Title updated for now will change date for actual release.

**Comment by Valastiri** [comment]
Boris suggested removing the copy about events added but no logic. I'd say leave it in and let the reader parse that at will. Will leave that up 2 final review.

**Review by ErickSharp — APPROVED** [review]

**Pull request merged**